src/main/java/sqlline/Commands.java

```diff
@@ -947,7 +948,22 @@ private void execute(String line, boolean call, DispatchCallback callback) {
 
       try {
         long start = System.currentTimeMillis();
-
+        String pattern = sqlLine.getOpts().getConfirmPattern();
+        if (Objects.equals(BuiltInProperty.CONFIRM_PATTERN
```

Comment:
This logic is better to move into setter of confirmpattern property is described at https://github.com/julianhyde/sqlline/pull/263/files#r248554757 .
And here should not be checks if the confirmPattern is default or not (everything should be done while setting value)